src/webots/gui/WbNewProjectWizard.cpp

```diff
@@ -48,7 +48,7 @@ QString WbNewProjectWizard::proposeNewProjectPath() const {
     path = WbPreferences::instance()->value("Directories/projects").toString();
     if (!path.isEmpty()) {
       if (WbFileUtil::isDirectoryWritable(path))
-        path += +"/my_project";
+        path += "my_project";
```

Comment:
Shouldn't we test if a folder separator is needed here?

Reply:
doesn't seem necessary as we set one every time: https://github.com/cyberbotics/webots/blob/3b4b6316f1545b3735ff9b04d5d203cd72ad7ccb/src/webots/gui/WbNewProjectWizard.cpp#L84, https://github.com/cyberbotics/webots/blob/3b4b6316f1545b3735ff9b04d5d203cd72ad7ccb/src/webots/editor/WbProjectRelocationDialog.cpp#L224 https://github.com/cyberbotics/webots/blob/3b4b6316f1545b3735ff9b04d5d203cd72ad7ccb/src/webots/core/WbPreferences.cpp#L108

which is why there were two `//`